cmd/xl-v1-utils_test.go

```diff
@@ -194,14 +196,15 @@ func getXLMetaBytes(totalParts int) []byte {
 func getSampleXLMeta(totalParts int) xlMetaV1 {
```

Comment:
Do we have a test-case for explicitly support the "old" format (where bit rot algorithm is (repeatedly) written alongside the checksum?)
(thus testing the `if` in line 180-182)